word-guess-main.rb

```diff
@@ -0,0 +1,86 @@
+
+
+class Game
+
+  def initialize(input_word, word_descriptions)
+    @word = input_word.chars.to_a
+    @hidden_word = @word.join.gsub(/[A-Za-z]/, "_")
+    @bad_guesses = 0
+    @word_descriptions = word_descriptions
+  end
+
+  def guess(letter)
+    if @word.include?(letter) == false
+      @bad_guesses += 1
```

Comment:
I really like that you make this its own method. It does one specific thing (handling a user's guess), and is just about the right amount of complexity to justify a method. Good work!